mono/utils/mono-dl-wasm.c

```diff
@@ -55,16 +55,21 @@ mono_dl_current_error_string (void)
 	return g_strdup ("");
 }
 
-
 int
 mono_dl_convert_flags (int flags)
 {
-	int lflags = flags & MONO_DL_LOCAL ? 0 : RTLD_GLOBAL;
+	int lflags = 0;
+
+	if (flags & MONO_DL_LOCAL)
+		flags |= RTLD_LOCAL;
+	else if (flags & MONO_DL_GLOBAL)
+		flags |= RTLD_GLOBAL;
```

Comment:
What is the expected behavior for specifying local and global?
Does this code have to be duplicated?

Reply:
Wasm currently doesn't use the rest of the posix stuff, so there's no file for me to share it in. It doesn't seem worth it to create a "posix + wasm" file just for this.